DataFormats/HcalDigi/src/classes_def.xml

```diff
@@ -42,11 +42,6 @@
    <class name="HcalTTPDigi" ClassVersion="10">
     <version ClassVersion="10" checksum="3127055275"/>
    </class>
-   <class name="HcalUpgradeDataFrame" ClassVersion="12">
-    <version ClassVersion="10" checksum="2143388425"/>
```

Comment:
It was only used for Phase2 before pre16. Since we only made relvals for those workflows, I don't think it will cause any problems. (I can add a drop statement if you tell me where it needs to go...)